src/orchestrator-process/src/lib.rs

```diff
@@ -350,6 +356,7 @@ impl NamespacedOrchestrator for NamespacedProcessOrchestrator {
         self.send_command(WorkerCommand::EnsureService {
             id: id.to_string(),
             config,
+            scheduling_config,
```

Comment:
Did you consider applying `always_use_disk` to the `ServiceConfig` here, instead of sending the whole `ServiceSchedulingConfig`? This looks like it would end up a bit less noisy, but more importantly it would be consistent with what we do in the k8s orchestrator.

Reply:
I cleaned up the code. Instead of sending the raw config and scheduling config, we send a custom struct. This only encodes the information we need downstream.